src/config/constants/farms.ts

```diff
@@ -12,6 +12,2431 @@ const farms: FarmConfig[] = [
+    lpSymbol: 'XRP-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x03F18135c44C64ebFdCBad8297fe5bDafdBbdd86',
+    },
+    token: tokens.xrp,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 264,
+    lpSymbol: 'USDT-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x16b9a82891338f9bA80E2D6970FddA79D1eb0daE',
+    },
+    token: tokens.usdt,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 263,
+    lpSymbol: 'ALPHA-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xACF47CBEaab5c8A6Ee99263cfE43995f89fB3206',
+    },
+    token: tokens.alpha,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 262,
+    lpSymbol: 'BTCB-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x61EB789d75A95CAa3fF50ed7E47b96c132fEc082',
+    },
+    token: tokens.btcb,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 261,
+    lpSymbol: 'ETH-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x74E4716E431f45807DCF19f284c7aA99F18a4fbc',
+    },
+    token: tokens.eth,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 260,
+    lpSymbol: 'XVS-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x7EB5D86FD78f3852a3e0e064f2842d45a3dB6EA2',
+    },
+    token: tokens.xvs,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 259,
+    lpSymbol: 'TWT-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x3DcB1787a95D2ea0Eb7d00887704EeBF0D79bb13',
+    },
+    token: tokens.twt,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 258,
+    lpSymbol: 'USDT-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0x7EFaEf62fDdCCa950418312c6C91Aef321375A00',
+    },
+    token: tokens.usdt,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 257,
+    lpSymbol: 'LINK-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x824eb9faDFb377394430d2744fa7C42916DE3eCe',
+    },
+    token: tokens.link,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 256,
+    lpSymbol: 'EOS-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xB6e34b5C65Eda51bb1BD4ea5F79d385Fb94b9504',
+    },
+    token: tokens.eos,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 255,
+    lpSymbol: 'DOT-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xDd5bAd8f8b360d76d12FdA230F8BAF42fe0022CF',
+    },
+    token: tokens.dot,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 254,
+    lpSymbol: 'BAND-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x168B273278F3A8d302De5E879aA30690B7E6c28f',
+    },
+    token: tokens.band,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 253,
+    lpSymbol: 'ADA-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x28415ff2C35b65B9E5c7de82126b4015ab9d031F',
+    },
+    token: tokens.ada,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 252,
+    lpSymbol: 'BUSD-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x58F876857a02D6762E0101bb5C46A8c1ED44Dc16',
+    },
+    token: tokens.busd,
+    quoteToken: tokens.wbnb,
+  },
+  /**
+   * All farms below here are from v1 and are to be set to 0x
```

Comment:
@hachiojidev  v1 farms got duplicated for some reason, we have them already down below.

Reply:
@Chef-Cheems Nice catch